state/graphql-sender/pom.xml

```diff
@@ -55,6 +55,10 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
```

Comment:
graphql-sender is used by the sdk - maybe better not to inflict the transitive dependency on users - or shade&relocate it.

To avoid the guava dep but keep the single-statement approach - another alternative might be something like:

```
new HashMap<String, Principal>() {{
  put("admin", Arrays.asList(new Principal("user1")));
  put("creator", Arrays.asList(new Principal("user2"), new Principal("user3")));
}}
```

Reply:
OK - good point. I am changing to this everywhere and removing the guava deps where I added them for consistency.